deploy/config/config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "tenantId": "a6da4e16-b382-4475-9ee1-1617bdda5b04",
+    "subscriptionId": "cd7267f6-3dfe-4cce-890e-bb35cc8c4bbc",
+    "cosmosReadUnitsDefault": 400,
```

Comment:
Agreed

Reply:
Deciding to leave this out of the config. This should need to change rarely, and if it's needed it can arguably be managed more easily in the cosmos UI than in config.